alembic/versions/0001.py

```diff
@@ -0,0 +1,534 @@
+"""0001
+
```

Comment:
@victor-nuzhniy i added new sequence and applied it to table
see more: https://github.com/anton-serozhechkin/e_university_api/blob/eu-release-0.2.0/migrations/main.sql#L33
pls check that this migration already applied in alembic migrations

Reply:
Everything works fine

Reply:
We don't need specify any sequence in postgres, everything works fine without it. And a better solution is Konstantin repo, and to implement it would be better after basic testing finished